spec/classes/ntp_spec.rb

```diff
@@ -143,6 +143,11 @@
             let(:params) {{ :package_ensure => 'present', :package_name => ['hambaby'] }}
             it { should contain_package('hambaby') }
           end
+
+          describe 'should allow the package to be unmanaged' do
+            let(:params) {{ :package_manage => false, :package_name => ['ntp'], }}
+            it { should_not contain_package('ntp') }
```

Comment:
Since the previous test installed the package, it will be installed, and package_manage => false does not uninstall the package, so this test will fail.